airflow/providers/amazon/aws/operators/step_function.py

```diff
@@ -42,6 +45,11 @@ class StepFunctionStartExecutionOperator(BaseOperator):
     :param state_machine_input: JSON data input to pass to the State Machine
     :param aws_conn_id: aws connection to uses
     :param do_xcom_push: if True, execution_arn is pushed to XCom with key execution_arn.
+    :param waiter_max_attempts: Maximum number of attempts to poll the execution.
+    :param waiter_delay: Number of seconds between polling the state of the execution.
+    :param deferrable: If True, the operator will wait asynchronously for the job to complete.
+        This implies waiting for completion. This mode requires aiobotocore module to be installed.
+        (default: False)
```

Comment:
```suggestion
```
this default is not exactly true anymore

Reply:
Thanks, updated the docstring